py3status/modules/coin_market.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+"""
+Display cryptocurrency data from https://coinmarketcap.com
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A message from the site:
+        Please limit requests to no more than 10 per minute.
+        (default 600)
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins
+        (default '[\?color=#ff0 {name}] ${price_usd:.2f} {percent_change_24h}')
```

Comment:
if it was at least a named color then the user could change it easily but I think it would still be hard to implement cleanly